src/site/content/en/blog/idle-detection/index.md

```diff
@@ -163,6 +163,21 @@ controller.abort();
 console.log('IdleDetector is stopped.');
 ```
 
+### DevTools support
+
+Starting in Chrome&nbsp;86, you can emulate idle events in the Chrome DevTools, no need to actually idle around.
```

Comment:
```suggestion
Starting in Chrome&nbsp;86, you can emulate idle events in Chrome DevTools without needing to idle around.
```

Reply:
What does 'idle around' mean? I get what you're saying. What I'm having trouble with is that 'idle' is a lack of action, yet you're  treating it like an action. I wonder if there's a better way to say that.

Reply:
"you can emulate idle events in Chrome DevTools without actually being idle."

The reverse were amazing: you can emulate busy events in Chrome DevTools without actually being busy. :-)